hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSQueueConverter.java

```diff
@@ -182,17 +184,18 @@ public void testQueueMaxAMShare() {
 
     // root.admins.bob
     assertEquals("root.admins.bob AM share", "1.0",
-        csConfig.get(PREFIX + "root.admins.bob.maximum-am-resource-percent"));
+        capacitySchedulerConfig.get(
```

Comment:
getMaximumAMResourcePercentPerPartition checks this correctly if queried for No Label label.